CMakeLists.txt

```diff
@@ -175,53 +175,13 @@ link_directories(
     ${LLVM_LIBRARY_DIRS}
 )
 
-if (LLVM_LINK_LLVM_DYLIB)
-  set (LLVM_COMPONENTS LLVM)
-else (LLVM_LINK_LLVM_DYLIB)
-  set (LLVM_COMPONENTS
-    Analysis
-    AsmParser
-    AsmPrinter
-    BitReader
-    BitWriter
-    CodeGen
-    Core
-    IRReader
-    InstCombine
-    Instrumentation
-    MC
-    MCDisassembler
-    MCParser
-    ObjCARCOpts
-    Object
-    Option
-    ProfileData
-    ScalarOpts
-    SelectionDAG
-    Support
-    Target
-    TransformUtils
-    Vectorize
-    X86AsmParser
-    X86AsmPrinter
-    X86CodeGen
-    X86Desc
-    X86Disassembler
-    X86Info
-    X86Utils)
-endif (LLVM_LINK_LLVM_DYLIB)
-
 set(ADDITIONAL_LIBS )
 
-if(USE_PREBUILT_LLVM AND NOT LLVMSPIRV_INCLUDED_IN_LLVM)
```

Comment:
This will cause linker error for LLVMSPIRVLib for systems that have LLVMSPIRV built into libLLVM.so